ui/src/main/resources/Meeting/Code/NextMeetingsMacro.xml

```diff
@@ -371,19 +371,27 @@
 #set($discard = $xwiki.ssx.use('Meeting.Code.NextMeetingsMacro'))
 #set($limit = $xcontext.macro.params.limit)
 #set($nowDate = $datetool.date)
-#set($query = "from doc.object(Meeting.Code.MeetingClass) as meeting where doc.fullName &lt;&gt; 'Meeting.Code.MeetingTemplate' and meeting.startDate &gt; :date and meeting.status in ('value1','value2') order by meeting.startDate asc")
+
+#set($displayOnGoingMeetings = $xcontext.macro.params.displayOnGoingMeetings)
+#set ($queryDateType = 'endDate')
+#if ($displayOnGoingMeetings == 'false')
+  #set ($queryDateType = 'startDate')
+#end
+
+#set($query = "from doc.object(Meeting.Code.MeetingClass) as meeting where doc.fullName &lt;&gt; 'Meeting.Code.MeetingTemplate' and meeting.${queryDateType} &gt; :date and meeting.status in ('value1','value2') order by meeting.startDate asc")
 #set($results = $services.query.xwql($query).setLimit($limit).bindValue('date', $nowDate).execute())
 ##
 (% class="meetings-next" %)(((
   #foreach($item in $results)
     #set($docReference = $services.model.resolveDocument($item))
     #set($document = $xwiki.getDocument($item))
+    #set($startDate = $document.getValue('startDate'))
     #if($services.security.authorization.hasAccess('view', $docReference))
       (% class="list-unstyled" %)
         #if($xcontext.macro.params.status == 'show')
           #set($status = "- $services.localization.render('contrib.meeting.field.status') : $document.display('status', 'view')")
         #end
-        * $services.icon.render('time')(% class="text-muted" %) [[$document.plainTitle&gt;&gt;$item]] $document.display('startDate','view') $!status
+        * $services.icon.render('time')(% class="text-muted" %) [[$document.plainTitle&gt;&gt;$item]] $document.display('startDate','view') $!status #if ($startDate.compareTo($nowDate) &lt; 0)($escapetool.xml($services.localization.render('meeting.panel.activeMeeting')))#end
```

Comment:
You're not inside an HTML macro so XML escaping doesn't make sense. You have two options:
* escape wiki syntax, see https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-rendering/xwiki-platform-rendering-xwiki/src/main/java/org/xwiki/rendering/script/RenderingScriptService.java#L216
* or, even simpler, use the translation macro https://extensions.xwiki.org/xwiki/bin/view/Extension/Translation%20Macro .